middlewares/body-validation/schema/blockAnonymousCommentV2Schema.js

```diff
@@ -0,0 +1,9 @@
+const Schema = {
+  source: 'string',
+  postId: 'string',
+  commentId: 'string',
+  reason: 'array|optional:true',
+  message: 'string|optional:true'
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The schema for `blockAnonymousCommentV2` seems to be well defined. However, it's not clear what the `reason` field is supposed to contain. If it's an array of strings, it should be explicitly mentioned as 'array:string'. Also, if `commentId` is optional, it should be marked as such.

```diff
-  commentId: 'string',
+  commentId: 'string|optional:true',
-  reason: 'array|optional:true',
+  reason: 'array:string|optional:true',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->